### PR TITLE
This adds Ordnance Survey's Places API connection artefacts

### PR DESCRIPTION
**Pull request by quickjc1**
---
Please check the following conditions for your PR.

- [x] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [x] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

### CODE REVIEW

File: certified-connectors/Ordnance Survey Places/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,2985 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "OS Places",
+    "description": "The OS Places API provides a detailed view of an address and its life cycle. It contains all the records of AddressBase\u00ae Premium and so provides all the information relating to an address or property from creation to retirement. It contains local authority, Ordnance Survey and Royal Mail\u00ae addresses, current addresses, and alternatives for current addresses, provisional addresses (such as planning developments) and historic information.",
```
Comment (sriyen-msft) [inline certified-connectors/Ordnance Survey Places/apiDefinition.swagger.json:6]: @quickjc1 Can you please remove unicodes from the description and if you have any in the swagger or apiProperties.josn file as unicodes may not be supported in all the Power Automate pages.

File: certified-connectors/Ordnance Survey Places/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,2985 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "OS Places",
```
Comment (sriyen-msft) [inline certified-connectors/Ordnance Survey Places/apiDefinition.swagger.json:5]: @quickjc1 The title `"OS Places"` looks generic name, Can you please rename it to `Ordnance Survey's Places` ?

File: certified-connectors/Ordnance Survey Places/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,2985 @@
...
+                      }
+                    }
+                  },
+                  "description": "results"
+                }
+              }
+            }
+          }
+        },
+        "summary": "OS Places API Find Operation",
```
Comment (sriyen-msft) [inline certified-connectors/Ordnance Survey Places/apiDefinition.swagger.json:363]: @quickjc1 This summary value can be improved, something like below:

`Find Operation`

Applies to all the similar summary values in the swagger

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Review by dougelliott — DISMISSED** [review]
Approved, pending Srikanth's feedback

File: certified-connectors/Ordnance Survey Places/apiDefinition.swagger.json (hunk 1 above)
Comment (quickjc1) [reply certified-connectors/Ordnance Survey Places/apiDefinition.swagger.json:6]: @sriyen-msft yes no problem

File: certified-connectors/Ordnance Survey Places/apiDefinition.swagger.json (hunk 2 above)
Comment (quickjc1) [reply certified-connectors/Ordnance Survey Places/apiDefinition.swagger.json:5]: @sriyen-msft agreed, will update

File: certified-connectors/Ordnance Survey Places/apiDefinition.swagger.json (hunk 3 above)
Comment (quickjc1) [reply certified-connectors/Ordnance Survey Places/apiDefinition.swagger.json:363]: @sriyen-msft agreed, will change

**Review by dougelliott — APPROVED** [review]

**Pull request merged**